desktop/core/src/desktop/js/utils/hooks/useFileUpload/util.ts

```diff
@@ -34,8 +39,24 @@ export interface UploadMetaData {
   qquuid: string;
 }
 
-export interface UploadChunkItem extends UploadItem {
+export interface UploadChunkItem extends Omit<UploadItem, 'file'> {
```

Comment:
Can we get a comment explaining what an UploadChunkItem is. A file, a part of a file, multiple parts of a file?